src/MetaMorpho.sol

```diff
@@ -737,6 +738,8 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         MarketConfig storage marketConfig = config[id];
 
         if (supplyCap > 0) {
+            if (config[id].removableAt != 0) revert ErrorsLib.PendingRemoval();
```

Comment:
we don't need this as it is checked in submit cap

Reply:
Yes but if I remember well we wanted to also have it here as a security.

Reply:
hmm feels weird

Reply:
@Jean-Grimal can you update the PR please?

Reply:
pushed